deploy/crds/examples/integreatly-rhmi-cr.yaml

```diff
@@ -11,3 +11,4 @@ spec:
   smtpSecret: INSTALLATION_PREFIX-smtp
   deadMansSnitchSecret: INSTALLATION_PREFIX-deadmanssnitch
   pagerDutySecret: INSTALLATION_PREFIX-pagerduty
+  pullSecret: {"name": "DELOREAN_PULL_SECRET_NAME","namespace": "DELOREAN_PULL_SECRET_NAMESPACE"}
```

Comment:
We shoudl change this to yaml. I'm not sure why you said it didn't work, but i don't think we need a mix of yml and json in here.